extension/observer/ecstaskobserver/extension.go

```diff
@@ -0,0 +1,47 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ecstaskobserver
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer"
+)
+
+var _ component.Extension = (*ecsTaskObserver)(nil)
+var _ observer.Observable = (*ecsTaskObserver)(nil)
+
+type ecsTaskObserver struct {
+	logger *zap.Logger
+	config *Config
+}
+
+func (e *ecsTaskObserver) Start(_ context.Context, host component.Host) error {
+	return nil
+}
```

Comment:
use https://pkg.go.dev/go.opentelemetry.io/collector@v0.38.0/component/componenthelper to build the extension so no need to implement funcs like Start/Shutdown if not needed.